File: 1-Two-Sum/1-two-sum.cpp
#include <iostream>
#include <vector>

class Solution
{
public:
  std::vector<int> twoSum(std::vector<int> &nums, int target)
  {
    std::vector<int> result;

    for (int i = 0; i < nums.size(); i++)
    {
      for (int j = i + 1; j < nums.size(); j++)
      {
        if (nums.at(i) + nums.at(j) == target)
        {
          result.push_back(i);
          result.push_back(j);
          return result;
        }
      }
    }

    return result;
  }
};

File: leetcode/1-two-sum/1-two-sum.py
class Solution(object):
  def twoSum(self, nums, target):
    """
    :type nums: List[int]
    :type target: int
    :rtype: List[int]
    """

    arrayLength = len(nums)
    rtype = []
    
    for i in range(0, arrayLength):
      for j in range (i + 1, arrayLength):
        if (nums[i] + nums[j] == target):
          rtype.append(i)
          rtype.append(j)
          return rtype